bower.json

```diff
@@ -83,6 +81,12 @@
     "url-search-params": {
       "main": "build/url-search-params.js"
     },
+    "font-awesome": {
```

Comment:
``font-awesome`` seems to be addes twice now (I switched to Font Awesome in b108efae5c439ce86b0c3c5b0b361ad96f5a5db6 for v0.6.0)?